Base/BacnetPropetyState.cs

```diff
@@ -20,12 +22,30 @@ public enum BacnetPropertyStateTypes
             LIFE_SAFETY_STATE
         }
 
+        public struct State
```

Comment:
why create a nested struct and not just create the fields directly? in future I would like to avoid having those massive structs will most fields empty depending on the use case, but this is how the original code was written so for now we stick to it.

Reply:
Yes, I certainly agree with this.

Besides, the default values are also confusing since it looks like they contain a value if you don't know what's going on underneath.